Common/itkComputeJacobianTerms.hxx

```diff
@@ -239,10 +237,6 @@ ComputeJacobianTerms<TFixedImage, TTransform>::Compute(double & TrC, double & Tr
   }
   for (const auto & sample : *sampleContainer)
   {
-    /** Print progress 0-50%.
-     *progressObserver->UpdateAndPrintProgress( samplenr );*/
-    ++samplenr;
```

Comment:
@mstaring The progressObserver calls in itkComputeJacobianTerms have been commented out from the very first itkComputeJacobianTerms, as far as I can see, commit e9de4b1281a07d2fdbaa96db865f3c7aaaf4e5d2 almost 8 years ago. Is it OK if I just remove them now? The helper variable `samplenr` is now unused, and caused compiler warnings.

Reply:
For the record, Marius just double-checked, and thought this commit should be OK 👍